web/tests/composebox_typeahead.test.cjs

```diff
@@ -892,7 +894,7 @@ test("content_typeahead_selected", ({override}) => {
     ct.get_or_set_token_for_testing("");
     actual_value = ct.content_typeahead_selected(
         {
-            topic: "Sweden",
+            topic_display_name: "Sweden",
```

Comment:
Let's add a test for `topic_display_name: REALM_EMPTY_TOPIC_DISPLAY_NAME `

You can look at `REALM_EMPTY_TOPIC_DISPLAY_NAME ` in other test files.